lib/src/components/NxSidebarNavigation/NxSidebarNavigation.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { faBars, faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+
+import NxButton from '../NxButton/NxButton';
+import NxCloseButton from '../NxCloseButton/NxCloseButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, SidebarNavLinkProps, propTypes } from './types';
+
+import './NxSidebarNavitation.scss';
+
+const NxSidebarNavLink = function NxSidebarNavLink({ name, href, icon, current }: SidebarNavLinkProps) {
+  const navigationLinkClassnames = classnames('nx-sidebar-nav--item', {
+    'selected': current
+  });
+
+  return (
+    <li className={navigationLinkClassnames} key={`nav-link-${name}`}>
```

Comment:
I don't see any reason why this needs to be a list. `<a>`s wrapped with `<nav>` should be sufficient and better for accessibility.

Reply:
> I don't see any reason why this needs to be a list. `<a>`s wrapped with `<nav>` should be sufficient and better for accessibility.

Done in 81e0139d95d4864157bcc88e144d66c34ba93f81